src/Layers/xrRender_R2/r2_R_sun_support.h

```diff
@@ -216,7 +216,7 @@ class FixedConvexVolume
             float min_dist = 2 * map_size;
             for (int p = 0; p < 4; ++p)
             {
-                float dist;
+                float dist = 0;
```

Comment:
This is better:
```suggestion
                float dist = 0.f;
```
or
```suggestion
                float dist = 0.0f;
```